### DISCUSSION TITLE
Welcome to raylib Discussions!

### DISCUSSION THREAD

**Discussion by raysan5** [raysan5/raylib#1453] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AregevDev** [comment]
Wow, that's cool!

**Reply by MartinLeidel** [reply]
So great! :)

**Comment by code5717** [comment]
Join Discord https://discord.gg/mzCY3wN

**Reply by raysan5** [reply]
Oh! Nice! Thanks for sharing Discord link!

**Comment by bogdanbankov** [comment]
Seems good

**Comment by Faiz12002** [comment]
Hi, I want to learn raylib.
It seems I didn't get any help from discord guys, maybe in this GitHub conversation I may get some help.

**Reply by raysan5** [reply]
Actually Discord is the best place to ask for help. But what problem do you have?

**Comment by Faiz12002** [comment]
I joined the discord server days ago, then downloaded raylib, I use vscode,
well, i could build a project (no console errors, and i used the preloaded
C file for testing) the exe file was created but when executing it, it
shows me a message box saying "Assertion failed", and that's the problem.
-> In discord server I asked in " setup help" section, I'm Fai with Samus
photo.

**Comment by tahoemike** [comment]
Thanks for providing the windows installer (raylib_installer_v300.mingw.exe).   However, the batch files for compiling the examples use "C:\Github\raylib..." but the installer creates directory "C:\Raylib..." so they don't work.  Also the "set RAYLIB_RES_FILE" command specifies C:\Github\raylib\raylib.rc.data but should be "C:\raylib\raylib\src\raylib.rc.data".  After editing the batch files everything worked great!

**Reply by raysan5** [reply]
That will be reviewed for the next release...

**Comment by Urbaner3** [comment]
Here I would like to give my opinion as developing rule. Better to keep the whole project work once it is in the stated provided. Based on my experience it is not the case, while [starter project](https://github.com/educ8s/Raylib-CPP-Starter-Template-for-VSCODE) gets to work in the video. I am going to test on it and [Andrew](https://www.youtube.com/watch?v=xWWqhQ1JnvE&t=72s)  which modified makefile content with name and source directories. Plus, he uses snippet, which makes it easier on editing code pages.
